app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    with open(parts[1], "r") as file_in, open(parts[2], "w") as file_out:
+        if parts[1] != parts[2]:
+            file_out.write(file_in.read())
```

Comment:
What is happen, if you get input not "cp" command?
![image](https://github.com/user-attachments/assets/e53a6115-ea42-4277-b147-cfcc03a1b321)